client/webpack.client.base.config.js

```diff
@@ -12,7 +12,10 @@ const cssVariables = require('./app/assets/styles/variables');
 
 const devBuild = process.env.NODE_ENV !== 'production';
 const nodeEnv = devBuild ? 'development' : 'production';
-const assetHost = typeof process.env.asset_host === 'string' ? `&asset_host=${process.env.asset_host}` : '';
+
+const { resolveAssetHost } = require('./webpackConfigUtil');
+const assetHostEnv = typeof process.env.asset_host === 'string' ? `&asset_host=${process.env.asset_host}` : '';
+const assetHost = resolveAssetHost(assetHostEnv);
```

Comment:
Does the random selecting now happen only once per build so that all asset paths will end up pointing to same host in one deployment, or do i get it wrong how webpack stuff here goes together?